server/index.js

```diff
@@ -66,7 +69,8 @@ app.get('/api/details', (req, res, next) => {
 
 app.post('/api/details/comment', (req, res, next) => {
   const { comment, gameId, rating } = req.body;
-  const userId = 1;
+  const user = jwtDecode(req.header('Authorization'));
```

Comment:
We don't use `jwtDecode` on the back end, instead create an authorization middleware that verifies the legitimacy of the token and assigns the extracted payload to `req.user`. `jwtDecode` still works even if the token is fraudulent.